systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/ConfigMapUtils.java

```diff
@@ -46,7 +48,44 @@ public static void waitForCreationOfConfigMap(String namespaceName, String confi
         TestUtils.waitFor(String.format("ConfigMap: %s/%s to be created", namespaceName, configMapName),
             TestConstants.POLL_INTERVAL_FOR_RESOURCE_READINESS,
             TestConstants.GLOBAL_TIMEOUT,
-            () -> kubeClient().getConfigMap(namespaceName, configMapName) != null
+            () -> getInNamespace(namespaceName, configMapName) != null
         );
     }
+
+    /**
+     * Returns {@link ConfigMap} from specified Namespace and with specified name.
+     *
+     * @param namespaceName     Namespace name where the ConfigMap should be present.
+     * @param configMapName     name of the desired ConfigMap.
+     *
+     * @return  {@link ConfigMap} from specified Namespace and with specified name.
```

Comment:
I am unsure if using the link this way is the proper way. I usually use it when I want to link some methods from different or current class f.e., `{@link class#methodOne()}`  I am not sure I see any value having just ` {@link ConfigMap}`. Maybe `{@link io.fabric8.kubernetes.api.model.ConfigMap} ` but that's too long IMO.

Reply:
You have it imported anyway, so when you use it like this, it's "clickable" and it will move you to particular class.
At least that's how it worked whole time for me across multiple PRs and I didn't have problem with that until that.